src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1116,6 +1116,10 @@ def dbapi(
         custom_schema: Optional[Union[str, StructType]] = None,
         predicates: Optional[List[str]] = None,
         session_init_statement: Optional[Union[str, List[str]]] = None,
+        imports: Optional[List[str]] = None,
+        packages: Optional[List[str]] = None,
+        external_access_integration: Optional[str] = None,
+        ingestion_mode: Literal["udtf_mode", "parquet_mode"] = "parquet_mode",
```

Comment:
I think the major different is one using client resource while the other using snowflake resource
something like client vs server?

cc: @sfc-gh-yixie

Reply:
I changed the name to 'local_mode' and 'cloud_mode', what do you think